helloworld/test/index-test.js

```diff
@@ -0,0 +1,22 @@
+'use strict';
+
+const chai = require('chai');
+const greet = require('../index.js');
+const assert = chai.assert;
+
+describe('test index.js', function(){
+  describe('test export with args', function(){
+    before(function(){
```

Comment:
Great job creating a ```before/after block``` to get used to the syntax. We'll be using it a lot more later. However, for the case of this test, this ```before/after block``` is unnecessary.

If you comment it out, your test still passes. That is because back in your ```index.js``` module, you lines 7-8 say:

```
var name = process.argv[2] || 'bangarang';
  return greet(name);
```

This makes it so that ```bangarang``` will ***always*** be the default name given no matter what if there is no ```procss.argv[2]``` argument. 

In other words, if I just type ```node index.js``` in my terminal, the default return value will be ```hello bangarang``` because of that ```||``` statement. If I type ```node index.js judy``` in my terminal, *then* I get ```hello judy``` because now we have a ```process.argv[2]``` argument provided. 

I hope this makes sense, ask me if you need more clarification.